src/js/Features.js

```diff
@@ -130,58 +130,57 @@ Features.prototype.isEnabled = function (featureName) {
 
     self._featureChanges = {};
 
-    var listElements = [];
+    const listElements = [];
+
+    for (let i = 0; i < self._features.length; i++) {
+        let feature_tip_html = '';
+        const rawFeatureName = self._features[i].name;
+        const featureBit = self._features[i].bit;
 
-    for (var i = 0; i < self._features.length; i++) {
-        var feature_tip_html = '';
         if (self._features[i].haveTip) {
-            feature_tip_html = '<div class="helpicon cf_tip" i18n_title="feature' + self._features[i].name + 'Tip"></div>';
+            feature_tip_html = `<div class="helpicon cf_tip" i18n_title="feature${rawFeatureName}Tip"></div>`;
         }
 
-        var newElements = [];
+        const newElements = [];
+
         if (self._features[i].mode === 'select') {
             if (listElements.length === 0) {
-                newElements.push($('<option class="feature" '
-                    + 'value="-1" '
-                    + 'i18n="featureNone" />'));
+                newElements.push($('<option class="feature" value="-1" i18n="featureNone" />'));
             }
-
-            var newElement = $('<option class="feature" id="feature-'
-                + i
-                + '" name="'
-                + self._features[i].name
-                + '" value="'
-                + self._features[i].bit
-                + '" i18n="feature' + self._features[i].name + '" />');
+            const newElement = $(` <option class="feature" id="feature-${i}" name="${rawFeatureName}" value="${featureBit}" i18n="feature${rawFeatureName}" />`);
 
             newElements.push(newElement);
             listElements.push(newElement);
         } else {
             let featureName = '';
             if (!self._features[i].hideName) {
-                featureName = `<td><div>${self._features[i].name}</div></td>`;
+                featureName = `<td><div>${rawFeatureName}</div></td>`;
             }
 
-            var newElement = $('<tr><td><input class="feature toggle" id="feature-'
-                    + i
-                    + '" name="'
-                    + self._features[i].name
-                    + '" title="'
-                    + self._features[i].name
-                    + '" type="checkbox"/></td><td><div>'
-                    + featureName
-                    + '</div><span class="xs" i18n="feature' + self._features[i].name + '"></span></td>'
-                    + '<td><span class="sm-min" i18n="feature' + self._features[i].name + '"></span>'
-                    + feature_tip_html + '</td></tr>');
+            const newElement = $(
+`<tr>
+  <td>
+    <input class="feature toggle" id="feature-${i}" "name="${rawFeatureName}" title="${rawFeatureName}" type="checkbox"/>
+  </td>
+  <td>
+    <div>${featureName}</div>
+    <span class="xs" i18n="feature${rawFeatureName}"></span>
+  </td>
+  <td>
+    <span class="sm-min" i18n="feature${rawFeatureName}"></span>
+    ${feature_tip_html}
+  </td>
+</tr>`
+            );
```

Comment:
This indentation is ugly, but I understand the reason to do this. I suppose we can try to include in a future pr the https://www.npmjs.com/package/common-tags package to work with html code easily.

Reply:
@McGiverGim In chrome devtools I see the ↵ character in innerHtml as in innerText. With template literals you have to start from column 0. Ultimately we should convert this is into Vue templates?

Reply:
Yeah, this is truly ugly.
An alternative to converting this to vue that is less involved is to stick the html as a hidden element into the .html file, and then use JQuery to clone it and add the right attributes.